docs/general/getting_started.md

```diff
@@ -15,6 +15,12 @@ And then execute:
 ```
 $ bundle
 ```
+### Available Commands
+
+|command                               |Description                                                 |
+|--------------------------------------|------------------------------------------------------------|
+| rails g resource + `your model`      | Create a new serializer is to generate a new resource.     |
```

Comment:
This should probably be a note that, by default, a serializer will be generated for every new resource.